src/Apps/IndustrialMetaverse/Site/KPIExplorer/index.ts

```diff
@@ -0,0 +1 @@
+//Not really sure what all needs to go here yet
```

Comment:
can we keep the same folder structure as we have in the LegionApp where it's got the Adapter, COmponent, Contexts folders under the root?

Reply:
![image](https://user-images.githubusercontent.com/57726991/235532220-8a27e517-f34e-45fb-9f16-b0e3c6637c3d.png)

Reply:
We have Legion and IM apps. IM only has Site (I could move it to an EntryPoints folder). Site has the two main "explorers" which could have Contexts/Adapters/Hooks of their own or have shared Adapters/Contexts/Hooks in Site. I definitely need to move any .tsx into Components folders and .types.ts into Models.